src/DSE.Open.Values/DateMonthOnly.cs

```diff
@@ -176,9 +176,9 @@ public bool TryFormat(Span<char> destination, out int charsWritten)
         return new(date);
     }
 
-    public static AgeInMonths operator -(DateMonthOnly a, DateMonthOnly b)
+    public static int operator -(DateMonthOnly a, DateMonthOnly b)
```

Comment:
The tests state that this should return the difference in months which is an `int`. Given this does not have much to do with age, we should return `int` not `AgeInMonths`.

Because `AgeInMonths` can no longer be negative, the test covering this case broke.

Reply:
Back to the discussion of `IInterval<TSelf, T>`.

We could decide AgeInMonths is just that a record of an age at a point in time (therefore constrained to sensible, non-negative values). And then we could have a `Months` unit of measure type that implements `IInterval<TSelf, T>`.

If we did this, then the difference between two `AgeInMonths` value could return a `Months` value.

For now, you will have to check if we use this arithmetic anywhere - we may well do.